src/vtbackend/Terminal.cpp

```diff
@@ -1562,11 +1562,26 @@ void Terminal::setWindowTitle(string_view title)
     _eventListener.setWindowTitle(title);
 }
 
+void Terminal::setTabName(string_view title)
```

Comment:
not used. either remove, or at least accept title as `std::string` and then move ;it into `_tabName` :)

Reply:
I removed this function